modules/payments/modules/payments/api.js

```diff
@@ -3,7 +3,7 @@ import { getGlobalOptions } from "@options";
 
 const global = getGlobalOptions(); 
 const BASE_URL = global.url; // change your BASE_URL in `options/options.js` to edit this value
-const token = "Token 676364403988909cde7f501aa2efeaf9ca30d18c"
+const token = "Token 913fdab66a6d73f3ed45a57a4fe6e8fa092fd153"
```

Comment:
Check my notes further below on how to access the token without having it exposed in the source code.